bin/start.js

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env node
 
+var log = require('npmlog')
+log.level = 'warn'
+
 var cli = require('../cli')
 
 var emoji = require('node-emoji')
 
 cli(function (error, server) {
   if (error) {
+    log.warn('DB url config', 'If you are using --dbUrl, --dbUrlUsername, --dbUrlPassword, you should provide your authentication details without URL encoding')
```

Comment:
There can be many errors that can returned by the `cli` callback, we cannot assume that is related to the DB url config. I think this is the wrong place to log this